islands/LgpdButton.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "preact/hooks";
+import { RichText } from "apps/admin/widgets.ts";
+
+interface Props {
+    text: RichText;
+    label: string;
+}
+
+export default function LgpdButton({ text, label }: Props) {
+    const [isAccepted, setIsAccepted] = useState(true)
+
+    const handleClick = () => {
+        localStorage.setItem('aceito', JSON.stringify(isAccepted))
+        setIsAccepted(false)
+    }
+
+    const verify = localStorage.getItem('aceito')
+
+    useEffect(() => {
+        setIsAccepted(false)
+        verify === 'true' ? setIsAccepted(false) : setIsAccepted(true)
```

Comment:
Estas definições de estados estão muito mais rebuscadas que o necessário.
```suggestion
        setIsAccepted(verify === 'true' )
```
Conforme outra sugestão, vamos **inverter** o `true` para ser quando _não_ será exibido, e `false` para quando será exibido, já que indica que o popup foi aceito ou não pelo usuário.